app/helpers/reports/regions_helper.rb

```diff
@@ -3,4 +3,8 @@ def percentage_or_na(value, options)
     return "N/A" if value.blank?
     number_to_percentage(value, options)
   end
+
+  def cohort_report_type(period)
+    "#{period.type.to_s.humanize}ly report"
```

Comment:
Not sure we need the "report" verbage in here, should we leave that up to the view?

Reply:
This is just a view helper, so I think this is fine.